docs/src/components/landing/Hero.js

```diff
@@ -43,24 +44,10 @@ export default function Hero() {
           Build scalable and secure internal tools locally. Use your own IDE, drag and drop
           pre-built components or create yours.
         </Typography>
-        <Typography
-          component="label"
-          variant="body2"
-          color={(theme) => (theme.palette.mode === 'dark' ? '#fff' : `text.secondary`)}
-          sx={{ fontWeight: 'medium', display: 'block', mb: 1, mx: 'auto' }}
-          htmlFor="email-landing"
-        >
-          Sign up for early access to the beta release.
-        </Typography>
-        <SignUp
-          sx={{
-            '& > div': {
-              display: 'flex',
-              flexDirection: { xs: 'column', sm: 'row' },
-              justifyContent: 'center',
-              maxWidth: 'initial',
-            },
-          }}
+        <GetStartedButtons
+          sx={{ justifyContent: 'center' }}
+          installation={'npx create-toolpad-app my-toolpad-app'}
```

Comment:
```suggestion
          installation={'npx create-toolpad-app'}
```

I suppose we don't want to enforce a name on the user?